src/index.jsx

```diff
@@ -1,9 +1,11 @@
 import React from 'react';
 import ReactDOM from 'react-dom';
+import { Provider } from 'react-redux';
 
+import store from './store';
 import App from './App';
 
 ReactDOM.render(
-  <App />,
+  <Provider store={store}><App /></Provider>,
```

Comment:
중첩된 컴포넌트 구조는 줄바꿈 처리를 해주는게 더 가독성에 좋을 것 같습니다~

```suggestion
  <Provider store={store}>
    <App />
  </Provider>,
```